addons/loadout/functions/fnc_getWeaponList.sqf

```diff
@@ -0,0 +1,42 @@
+#include "script_component.hpp"
+/*
+ * Author: NeilZar
+ * Get all turret weapons from the passed vehicle, the compatible magazines, and a count of each magazine currently in the turret.
+ *
+ * Arguments:
+ * 0: Vehicle <OBJECT>
+ *
+ * Return Value:
+ * Array of all weapons and their magazines <ARRAY>
+ *
+ * Example:
+ * [_vehicle] call zen_loadout_fnc_getWeaponList
+ *
+ * Public: No
+ */
+
+params ["_vehicle"];
+
+private _pylons = (getPylonMagazines _vehicle) apply {toLower (getText (configFile >> "CfgMagazines" >> _x >> "pylonWeapon"))};
```

Comment:
`_pylonWeapons` maybe. Also, can store the "CfgMagazines" lookup like "CfgWeapons". No need for parentheses after `toLower` or around `getPylonMagazines`.